service/src/main/java/uk/gov/hmcts/reform/fpl/NotifyTemplates.java

```diff
@@ -11,7 +11,8 @@ private NotifyTemplates() {
     public static final String GATEKEEPER_SUBMISSION_TEMPLATE = "a79cffc8-7aec-445c-b310-7bed5fb479bc";
     public static final String STANDARD_DIRECTION_ORDER_ISSUED_TEMPLATE = "c06968ea-048b-49d6-a4b5-1f13ee995ba1";
     public static final String C2_UPLOAD_NOTIFICATION_TEMPLATE = "6b961e81-c5ff-4f6f-8c56-f90d932a2f9b";
-    public static final String ORDER_NOTIFICATION_TEMPLATE = "1f7c134e-f9c0-44ba-aa50-fce53eb208f7";
+    public static final String ORDER_NOTIFICATION_TEMPLATE_FOR_LA = "1f7c134e-f9c0-44ba-aa50-fce53eb208f7";
+    public static final String ORDER_NOTIFICATION_TEMPLATE_FOR_ADMIN = "c652061a-31db-4ec5-bcf7-4acec5943658";
     public static final String CMO_ORDER_ISSUED_CASE_LINK_NOTIFICATION_TEMPLATE =
```

Comment:
please could you confirm the `cmoCallout` actually displays as a callout once the email is sent?
<img width="660" alt="Screenshot 2020-02-14 at 10 25 10" src="https://user-images.githubusercontent.com/6439798/74523102-69220980-4f14-11ea-9d8a-4bc7142d0c4c.png">

Reply:
<img width="1165" alt="image" src="https://user-images.githubusercontent.com/15530123/74526621-a8078d80-4f1b-11ea-8c7a-6ae78fd38ff2.png">